app/views/ncr/work_orders/_attachments.html.haml

```diff
@@ -0,0 +1,6 @@
+.form-group
+  = field_set_tag 'Attachments' do
+    %ul.attachments{ data: { add_minus: true } }
+      - 10.times do
```

Comment:
I believe it _is_ an arbitrary number...correct @cmc333333? https://github.com/18F/C2/pull/380 Moving to a constant.